### DISCUSSION TITLE
Inferring trajectory results

### DISCUSSION THREAD

**Discussion by Umaarasu** [aertslab/scenicplus#356] [General]
Hi @SeppeDeWinter ,

In your article there is a heatmap (figure 6: https://www.biorxiv.org/content/10.1101/2022.08.19.504505v1.full.pdf) showing standardized TF expression (top) and regulon AUC values (bottom) along the oligodendrocyte differentiation pseudotime. 

![Näyttökuva 2024-04-16 121011](https://github.com/aertslab/scenicplus/assets/86348447/e0ec56e6-2122-484f-a73b-56ca6851b4d8)

Why is the TF expression (top) along the trajectory is bit rough while the AUC values (bottom) has a smooth spline along the trajectory?

**Comment by SeppeDeWinter** [comment]
Hi @Umaarasu 

The AUC value is a measurement of multiple genes (i.e. all predicted target genes of a certain TF) while the TF expression is just a single value. I believe this is the reason for the AUC values being more smooth. By combining the values of multiple genes it will smooth the values out a bit.

Best,

Seppe

**Reply by Umaarasu** [reply]
@SeppeDeWinter Thank you :)
So, there is this scenario now where I can see the TF in the eRegulon metadata that regulates my gene. But the gene is not present in the list of genes regulated by that TF.  For example; literature says GeneX is regulated by TF1. I can see TF1 in my eRegulon metadata. But GeneX is not there. I can see the expression of GeneX in scRNA seq data.  How to decipher the result in this scenario.

**Reply by SeppeDeWinter** [reply]
Hi

There could be multiple reasons:
- the gene is not co-expressed with the TF (check the TF to gene adjacencies in this case)
- the regions that are near the gene don't have the motif for the TF (check the motif enrichment results for this case)

All the best,

Seppe

**Reply by Umaarasu** [reply]
@SeppeDeWinter 
Thanks for your input. I had been going through the results based on your suggestions. The issue I mentioned came up when I performed trajectory analysis on a certain set of celltypes only. In the tutorial for the eRegulon metadata from the full SCENIC+ analysis feeds into the trajectory inference pipeline. Do you think it would be a option to run the TF to gene relationship and build eGRN for the subclustered cells in trajectory again? Could that take into account genes expressed in smaller clusters in smaller amounts and provide a different output on the eRegulon metadata?

**Reply by SeppeDeWinter** [reply]
Hi @Umaarasu 

Yes, the output will be different if you subset.

I general I don't suggest to subset your cells though, SCENIC+ relies on the background of other cells to be able to infer the GRN. But feel free to give it a try.

All the best,

Seppe

**Reply by Umaarasu** [reply]
@SeppeDeWinter Thanks :)
